contentcuration/contentcuration/frontend/shared/languageSwitcher/LanguageSwitcherList.vue

```diff
@@ -135,18 +129,7 @@
     text-align: left;
   }
 
-  .dots:not(:last-child)::after {
-    // because it is a pseudo-element, text-decoration none only works with 'display: inline-block`
-    display: inline-block;
-    margin: 0 8px;
-    font-size: 14pt;
-    color: var(--v-grey-base);
-    text-decoration: none;
-    vertical-align: middle;
-    content: '•';
-  }
-
-  .dotsSpan::after {
+  .dots :not(:first-child, :last-child)::after {
```

Comment:
We generally avoid writing css selectors that reach into other components-- `:not(:first-child, :last-child)` is applying to elements defined in `KButtonGroup`. The fact that we have to resort to this for a KDS component means it's sort of a deficiency in KDS. If _we_ try to avoid that, we should encourage that same principle for KDS components too. I haven't yet written up an issue for that.

Also, consider if another engineer revisits this component-- it isn't inherently clear that the css selector is applying to elements in `KButtonGroup`, which creates sort of an implicit dependency on its internal structure. I think it would be a good idea to mention in a code comment that what this is applying to.

Reply:
Yes, that is a good point about the css selector. Another solution with the same result is to wrap the second button in the `KButtonGroup` inside a span and apply the original dots class to it, making it clearer that the css selector only applies to that rendered list of `KButton` components. Do you think that would be a clearer solution?

Reply:
Actually, since it would still be helpful to include the code comment and refactoring the css selectors into one class has the added benefit of providing the desired results with less code, simply adding the comment to make this solution clearer makes sense.

Reply:
No! I think this is cleaner, but I think it's also motivating to figure out what to do in KDS. I'm thinking of a couple options: 1) there was a discussion at one point if we should integrate this into KDS, perhaps like a `delimiter` prop on the `kButtonGroup` since this is styling related and if we possibly want the same on Kolibri but I don't recall where that landed, 2) we could update `kButtonGroup` to do something like [Vuetify does](https://github.com/vuetifyjs/vuetify/blob/v1.5.0/packages/vuetify/src/components/VSelect/VSelectList.js#L180) to workaround this situation generally, perhaps check if a css class exists on the button group item, e.g. `button-group-item` (Vuetify checks if the component is a list item), and move the KDS [styling for that](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/buttons-and-links/KButtonGroup.vue#L35) to a global stylesheet, meaning we could apply the dots css explicitly in this Studio component

Reply:
That last reply was to your first comment

Reply:
I will followup with making improvements to KDS